test_scripts/API/KeyboardEnhancements/Capabilities/005_HMI_provides_invalid_value_for_one_parameter_in_KeyboardCapabilities.lua

```diff
@@ -0,0 +1,62 @@
+----------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0238-Keyboard-Enhancements.md
+----------------------------------------------------------------------------------------------------
+-- Description: Check SDL is unable to provide 'WindowCapabilities' to App
+-- in case if HMI has sent 'OnSystemCapabilityUpdated' notification with invalid data
+--
+-- Steps:
+-- 1. App is registered
+-- 2. HMI provides 'KeyboardCapabilities' with invalid data within 'OnSystemCapabilityUpdated' notification
+-- 3. App requests 'DISPLAYS' system capabilities through 'GetSystemCapability'
+-- SDL does:
+--  - Respond with DATA_NOT_AVAILABLE, success:false to App
+----------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/KeyboardEnhancements/common')
+
+--[[ Local Variables ]]
+local keyboardLayouts = { "QWERTY", "QWERTZ", "AZERTY", "NUMERIC" }
+
+local tcs = {
+  [01] = { maskInputCharactersSupported = "false" },
```

Comment:
```suggestion
  [01] = { maskInputCharactersSupported = "false" }, --invalid type
```
I propose to add a description for all mentioned cases  e.g. : upper out of bound or invalid type